.packit.yml

```diff
@@ -9,7 +9,7 @@ actions:
   - >-
     bash -c '
     spectool
-    --define "upstream_version ${PACKIT_PROJECT_NAME_VERSION}"
```

Comment:
Oh, I probably did a bad copy-paste here. This is definitely not what I intended.